frontend/src/components/WidgetContainerTitleGroup.js

```diff
@@ -35,30 +49,67 @@ const WidgetContainerTitleGroup = ({
   ] : [];
 
   return (
-    <div className={`${showHeaderBorder ? 'border-bottom smart-hub-border-base-lighter' : ''} ${className} desktop:display-flex flex-justify flex-align-center flex-gap-2`}>
+    <div className={`smart-hub--table-widget-container ${showHeaderBorder ? 'border-bottom smart-hub-border-base-lighter' : ''} ${className} desktop:display-flex flex-justify flex-align-center flex-gap-2`}>
       <div className="desktop:display-flex flex-align-center flex-gap-2">
         <div>
-          <h2 className="smart-hub--table-widget-heading margin-0 margin-y-2 font-sans-lg">{title}</h2>
-          {subtitle ? <p className="usa-prose margin-x-0 margin-y-2">{subtitle}</p> : null }
+          <h2 className="smart-hub--table-widget-heading margin-0 margin-y-2 font-sans-lg">
+            {title}
+            {
+                titleDrawerText && (
+                <>
+                  <DrawerTriggerButton customClass="font-sans-lg margin-left-1 text-bold" drawerTriggerRef={titleDrawerRef}>
+                    {titleDrawerText}
+                  </DrawerTriggerButton>
+                  <Drawer
+                    triggerRef={titleDrawerRef}
+                    stickyHeader
+                    stickyFooter
+                    title={titleDrawerTitle}
+                  >
+                    <ContentFromFeedByTag tagName={titleDrawerCssClass} contentSelector="table" />
```

Comment:
probably not the best propname, this is not an HTML class but the tag we look up in the Jira API